test/test_trackable.py

```diff
@@ -124,3 +124,11 @@ def test_post_log(self, mock_request, mock_load_log_page):
                 "ctl00$ContentBody$LogBookPanel1$uxLogInfo": test_log_text,
             }
             mock_request.assert_called_with(self.t._log_page_url, method="POST", data=expected_post_data)
+
+    def test_load_KML(self):
```

Comment:
Please rename the test method according to the tested method. In this case `test_get_KML()`.